sample-app/src/Layout.tsx

```diff
@@ -0,0 +1,34 @@
+import Navigation from './components/Navigation';
+import SearchBar from './components/SearchBar';
+import { useAnswersState } from '@yext/answers-headless-react';
+import { universalResultsConfig } from './universalResultsConfig';
+import { LayoutComponent } from './PageRouter';
+
+const navLinks = [
+  {
+    to: '/',
+    label: 'All'
+  },
+  ...Object.entries(universalResultsConfig).map(([verticalKey, config]) => ({
+    to: verticalKey,
+    label: config.label || verticalKey
+  }))
+]
+
+/**
+ * A LayoutComponent that provides a SearchBar and Navigation tabs to a given page.
+ */
+const Layout: LayoutComponent = ({ page }) => {
```

Comment:
super nit: It might be easy to mix up the `Layout` here and the `Layout` in PageProps because one is a concrete implementation and the other simply obeys the `LayoutComponent` interface. What do you think about naming this `StandardLayout` in order to help differentiate them?

Reply:
yeah I like that! I also moved it to the `src/pages` folder